src/tools/pontoon/teams_projects.md

```diff
@@ -93,9 +93,9 @@ It’s possible to request an additional language for some projects from its *Pr
 
 ![Request a project](../../assets/images/pontoon/teams_projects/request_locale.png)
 
-Click on `REQUEST NEW LANGUAGE` (highlighted in red), select the language to add and then click `REQUEST NEW LANGUAGE`.
+Click on `REQUEST NEW LANGUAGE`, select the language to add and then click `REQUEST NEW LANGUAGE`.
```

Comment:
For a follow-up, we should consider if this should be called "REQUEST NEW LOCALE", since language is not really correct.

Reply:
Filed https://github.com/mozilla/pontoon/issues/2870